### PR TITLE
docs(common):UI for Blazor in Hybrid Apps

### PR DESCRIPTION
**Pull request by ntacheva**

### CODE REVIEW

File: hybrid-blazor-apps.md [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+You can now run the application in the expected for the technology stack way, following the corresponding instructions for how to run the app. Here are the related resources for ease of access:
+
+* [MAUI](https://docs.microsoft.com/en-us/dotnet/maui/get-started/first-app?pivots=devices-android)
+* [WinForms](https://docs.microsoft.com/en-us/visualstudio/ide/create-csharp-winform-visual-studio?view=vs-2022#run-the-application)
+* [WPF](https://docs.microsoft.com/en-us/dotnet/desktop/wpf/get-started/create-app-visual-studio?view=netdesktop-6.0#run-the-app)
+
+
+## Notes
+
+At the time of writing, this technology is in preview phase, and there may be difficulties and missing features. A few examples include:
```
Comment (Stamo-Gochev) [inline hybrid-blazor-apps.md:91]: This is no longer true as MAUI is now officially supported as well as the webviews.

File: hybrid-blazor-apps.md [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+
+* There is no debugging protocol exposed for the webview, so inspecting content and debugging is difficult.
+* Access to native APIs from the Blazor Web app code is still to be exposed by the framework - at the moment you have to write your own calls to services and code from the native app that you need to explicitly expose.
+* There are reports of difficulties getting MAUI to work. It is early days for it yet, and you need to ensure you can run it first, before adding Blazor to the mix.
```
Comment (Stamo-Gochev) [inline hybrid-blazor-apps.md:95]:
```suggestion
* There are reports of difficulties getting MAUI to work. It is early days for it yet, and you need to ensure you can run it first, before adding Blazor to the mix.
```
This is supposed to be fixed with the official release of MAUI.

File: hybrid-blazor-apps.md [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+position: 270
+---
+
+# UI for Blazor in Hybrid Apps
+
+The [WebView](https://devblogs.microsoft.com/dotnet/asp-net-core-updates-in-net-6-preview-3/#blazorwebview-controls-for-wpf-windows-forms) feature available as of .NET 6.0 now allows you to embed Blazor components into native MAUI, WPF and WinForms apps.
+
+This article provides details on how to setup the apps to use the Telerik UI for Blazor components.
+
+>tip Explore the [Hybrid Blazor Apps (Blazor Web apps running in WinForms, WPF, MAUI)](https://github.com/telerik/blazor-ui/tree/master/common/hybrid-blazor-apps) sample app.
```
Comment (dimodi) [inline hybrid-blazor-apps.md:17]: The samples apps are several. Consider two sentences instead of brackets.

File: hybrid-blazor-apps.md [hunk 4]
```diff
@@ -0,0 +1,100 @@
...
+
+* [Prerequisites](#prerequisites)
+* [Add the UI for Blazor components](#add-the-ui-for-blazor-components)
+* [Run the apps](#run-the-apps)
+* [Notes](#notes)
+
+
+## Prerequisites
+
+Prior to adding the Telerik components, you need to make sure you can run the corresponding technology stack and the basic Hybrid Blazor WebView in it.
```
Comment (dimodi) [inline hybrid-blazor-apps.md:30]: "you can run" may sound a bit condescending.

```suggestion
Prior to adding the Telerik components, ensure the corresponding technology stack is setup and the basic Hybrid Blazor WebView runs as expected in it.
```

File: hybrid-blazor-apps.md [hunk 5]
```diff
@@ -0,0 +1,100 @@
...
+You may add the Telerik resources as [static assets]({%slug getting-started/what-you-need%}#static-assets) or reference them from a [cloud CDN]({%slug getting-started/what-you-need%}#cdn).
+
+#### 3. Include `@using` statements 
+
+You can set the project to recognize all Telerik components without explicit `@using `statements on every `.razor` file. To achieve this, add the following to your `~/_Imports.razor`file:
+
+````
+@using Telerik.Blazor
+@using Telerik.Blazor.Components
+@using Telerik.Blazor.Services
```
Comment (dimodi) [inline hybrid-blazor-apps.md:61]: This line is not necessary in Blazor apps. Is the addition here on purpose? If yes, we can mention why.

File: hybrid-blazor-apps.md [hunk 6]
```diff
@@ -0,0 +1,100 @@
...
+* `Shared/MainLayout.razor` for MAUI apps
+* `Main.razor` for WPF and WinForms apps
+
+Make sure that the `TelerikRootComponent` matches the webview viewport. You might need to remove the default margin and padding from the body to ensure the content matches the viewport.
+
+Once custom layouts are supported, you will be able to configure a Telerik layout in the same way as with regular Blazor web apps (check [Common Configuration]({%slug getting-started/what-you-need%}#common-configuration)).
+
+#### 5. Add the UI for Blazor components
+
+After completing the above steps, you will be able to add your desired Telerik Blazor components in the app as in a native Blazor app. Explore the available UI for Blazor components and their features in our [live demos](https://demos.telerik.com/blazor-ui).
```
Comment (dimodi) [inline hybrid-blazor-apps.md:77]: We don't need "after completing..." in a numbered list. 
```suggestion
Add your desired Telerik Blazor components in the app as in a native Blazor app. Explore the available UI for Blazor components and their features in our [live demos](https://demos.telerik.com/blazor-ui).
```

File: hybrid-blazor-apps.md [hunk 7]
```diff
@@ -0,0 +1,100 @@
...
+
+
+## Run the apps
+
+You can now run the application in the expected for the technology stack way, following the corresponding instructions for how to run the app. Here are the related resources for ease of access:
```
Comment (dimodi) [inline hybrid-blazor-apps.md:82]: We mention that the developer should be familiar with the technology stack from the beginning, so this part might be unnecessary. Or we should move it up. 

```suggestion
You can now run the hybrid application. Refer to the following resources for each technology stack:
```

File: hybrid-blazor-apps.md [hunk 8]
```diff
@@ -0,0 +1,100 @@
...
+
+* [MAUI](https://docs.microsoft.com/en-us/dotnet/maui/get-started/first-app?pivots=devices-android)
+* [WinForms](https://docs.microsoft.com/en-us/visualstudio/ide/create-csharp-winform-visual-studio?view=vs-2022#run-the-application)
+* [WPF](https://docs.microsoft.com/en-us/dotnet/desktop/wpf/get-started/create-app-visual-studio?view=netdesktop-6.0#run-the-app)
+
+
+## Notes
+
+A couple points to consider:
+
```
Comment (dimodi) [inline hybrid-blazor-apps.md:92]:
```suggestion
```

File: hybrid-blazor-apps.md [hunk 9]
```diff
@@ -0,0 +1,100 @@
...
+* There is no debugging protocol exposed for the webview, so inspecting content and debugging is difficult.
```
Comment (dimodi) [inline hybrid-blazor-apps.md:93]: Do we have a suitable link here? If not, it's better to remove the second part of the sentence.

```suggestion
* There is no debugging protocol exposed for the webview, so inspecting content and debugging is difficult.
```

File: hybrid-blazor-apps.md [hunk 10]
```diff
@@ -0,0 +1,100 @@
...
+* Access to native APIs from the Blazor Web app code is still to be exposed by the framework - at the moment you have to write your own calls to services and code from the native app that you need to explicitly expose.
```
Comment (dimodi) [inline hybrid-blazor-apps.md:94]:
```suggestion
* The Blazor web app code cannot make calls to native APIs. This feature yet to be exposed by the framework. At the moment, you have to write your own calls to services and native app code that you need to explicitly expose.
```

File: hybrid-blazor-apps.md [hunk 11]
```diff
@@ -0,0 +1,100 @@
...
+* The WebView is not on the [list of officially supported browsers by Telerik UI for Blazor]({%slug browser-support%}). It has its specifics and differences from a standalone browser, and the hybrid blazor app integration should be considered a proof-of-concept for the time being. As the technology and framework matures, we will be monitoring it and we will consider adding it to the list of officially supported environments.
```
Comment (dimodi) [inline hybrid-blazor-apps.md:95]:
```suggestion
* The WebView is not on the [list of officially supported browsers by Telerik UI for Blazor]({%slug browser-support%}). It has its specifics and differences from a standalone browser. The hybrid Blazor app integration should be considered a proof-of-concept for the time being. We will monitor the framework maturity and consider adding the webview to the list of supported environments.
```

File: hybrid-blazor-apps.md [hunk 12]
```diff
@@ -0,0 +1,100 @@
...
+
+>caption Known issues:
+
+* Running MAUI apps might require developer mode to be [enabled](https://stackoverflow.com/questions/36324300/ensure-that-target-device-has-developer-mode-enabled-could-not-obtain-a-develop).
```
Comment (dimodi) [inline hybrid-blazor-apps.md:99]:
```suggestion
* Running MAUI apps may require [developer mode to be enabled](https://stackoverflow.com/questions/36324300/ensure-that-target-device-has-developer-mode-enabled-could-not-obtain-a-develop).
```

**Review by dimodi — CHANGES_REQUESTED** [review]
Use shorter sentences, less passive voice, and less subordinate clauses. Use correct heading hierarchy.

**Review by dimodi — APPROVED** [review]

**Pull request merged**